sympy/physics/vector/tests/test_point.py

```diff
@@ -126,3 +128,67 @@ def test_point_partial_velocity():
     assert p.partial_velocity(N, u1) == A.x
     assert p.partial_velocity(N, u1, u2) == (A.x, N.y)
     raises(ValueError, lambda: p.partial_velocity(A, u1))
+
+def test_point_vel():
+    q1, q2 = dynamicsymbols('q1 q2')
+    N = ReferenceFrame('N')
+    B = ReferenceFrame('B')
+    S = ReferenceFrame('S')
+    O = Point('O')
+    Q = Point('Q')
+    Q.set_pos(O, q1 * N.x)
+    raises(ValueError , lambda : Q.vel(N)) #Velocity of Q is not defined
+    O.set_vel(N, q2 * N.y)
+    assert Q.vel(N) == Derivative(q1) * N.x + q2 * N.y  # Velocity of Q using O
```

Comment:
```suggestion
    assert Q.vel(N) == q1.diff() * N.x + q2 * N.y  # Velocity of Q using O
```

Reply:
Derivative and diff both would differentiate wrt time then what's the difference

Reply:
It is an idiomatic difference.